### DISCUSSION TITLE
Release v0.19.1 Beta

### DISCUSSION THREAD

**Discussion by PrestonN** [FreeTubeApp/FreeTube#4117] [Announcements]
## Electron

* Electron has been updated to v22.3.25 which fixes the recent libwedp vulnerability

## Features
* Added external player support for the Clapper Media Player by @trostboot in https://github.com/FreeTubeApp/FreeTube/pull/3916
* Added Pastel & Hot Pink Themes by @jasonhenriquez in https://github.com/FreeTubeApp/FreeTube/pull/3937
* Implemented right-to-left interface for relevant languages by @jasonhenriquez in https://github.com/FreeTubeApp/FreeTube/pull/3970

## Fixes
* Fixed an issue where the video title was disappearing due to a YouTube page change by @PikachuEXE in https://github.com/FreeTubeApp/FreeTube/pull/4105
* Fixed the styling on the channels with errors list within the subscription view by @PikachuEXE in https://github.com/FreeTubeApp/FreeTube/pull/3921
* Fixed an issue where `Scroll playback rate over video player` prevented the page from scrolling when disabled by @PikachuEXE in https://github.com/FreeTubeApp/FreeTube/pull/3933
* Fixed an issue where clicking below the save icon wouldn't play the video by @vinit620 in https://github.com/FreeTubeApp/FreeTube/pull/3951
* Fixed an issue where raw HTML would show up in video descriptions by @jasonhenriquez in https://github.com/FreeTubeApp/FreeTube/pull/3946
* Fixed the styling of the watch page layout when theatre mode is enabled and sidebar is hidden by @PikachuEXE in https://github.com/FreeTubeApp/FreeTube/pull/3931
* Fixed an issue where the subscribe button clipped on the Channel page on tablet / mobile devices by @jasonhenriquez in https://github.com/FreeTubeApp/FreeTube/pull/4057
* Fixed an issue where buttons would intersect on the Subscribed Channels tab by @jasonhenriquez in https://github.com/FreeTubeApp/FreeTube/pull/4058
* Fixed an issue where playlists wouldn't always grab data when using the local API by @PikachuEXE in https://github.com/FreeTubeApp/FreeTube/pull/4102
* Fixed the styling for Channel page on desktop view by @jasonhenriquez in https://github.com/FreeTubeApp/FreeTube/pull/4112

## Changes
* Added the GitHub Discussions page to the Help section of the About tab by @efb4f5ff-1298-471a-8973-3d47447115dc in https://github.com/FreeTubeApp/FreeTube/pull/3901
* The search bar now recognizes youtube.com/live/ URLs by @PikachuEXE in https://github.com/FreeTubeApp/FreeTube/pull/3930
* The current resolution being used now displays when Auto is selected by @Benjababe in https://github.com/FreeTubeApp/FreeTube/pull/3935
* Video list buttons now only show up on hover or focus by @jasonhenriquez in https://github.com/FreeTubeApp/FreeTube/pull/3954
* New distraction free option to replace comment photos with generic ones by @Benjababe in https://github.com/FreeTubeApp/FreeTube/pull/3939
* Added a button to pause continuing the playlist after the current video by @Benjababe in https://github.com/FreeTubeApp/FreeTube/pull/3938
* Theatre mode will now be used by default if there's no sidebar by @absidue in https://github.com/FreeTubeApp/FreeTube/pull/3981
* Enable Persian language (Farsi) by @ChunkyProgrammer in https://github.com/FreeTubeApp/FreeTube/pull/4010
* Hashtags now show up in search results by @ChunkyProgrammer in https://github.com/FreeTubeApp/FreeTube/pull/3780
* Comments by channels that you're subscribed to now shows an indicator of your subscribed status by @Benjababe in https://github.com/FreeTubeApp/FreeTube/pull/4007
* The handling of watch history was given some optimizations and a performance boost by @absidue in https://github.com/FreeTubeApp/FreeTube/pull/4017
* The community tab has been added to the subscriptions tab by @ChunkyProgrammer in https://github.com/FreeTubeApp/FreeTube/pull/3973
* Added support for multiple audio tracks and AV1 for when using the Invidious API by @absidue in https://github.com/FreeTubeApp/FreeTube/pull/3942
* Added some visual improvements to the channel page for mobile and tablet devices by @jasonhenriquez in https://github.com/FreeTubeApp/FreeTube/pull/4061

## New Contributors
* @vinit620 made their first contribution in https://github.com/FreeTubeApp/FreeTube/pull/3951
* @Benjababe made their first contribution in https://github.com/FreeTubeApp/FreeTube/pull/3935
* @da-batt made their first contribution in https://github.com/FreeTubeApp/FreeTube/pull/3890
* @github-actions made their first contribution in https://github.com/FreeTubeApp/FreeTube/pull/3988
* @MaxMaxMaxCh made their first contribution in https://github.com/FreeTubeApp/FreeTube/pull/4060

**Full Changelog**: https://github.com/FreeTubeApp/FreeTube/compare/v0.19.0-beta...v0.19.1-beta

<hr /><em>This discussion was created from the release <a href='https://github.com/FreeTubeApp/FreeTube/releases/tag/v0.19.1-beta'>Release v0.19.1 Beta</a>.</em>

**Comment by senolfndk** [comment]
Can you add show comments in sidebar?

**Comment by gdwaynewarner** [comment]
I can't seem to get FreeTube to accept my YouTube subscriptions saved from Firefox.

The Mac used to be big on "Drag and Drop," which, of course, I tried, multiple times, to no avail.

How, exactly, does one get FreeTube to accept the downloaded files?

**Reply by ChunkyProgrammer** [reply]
Are you importing from youtube? https://docs.freetubeapp.io/usage/importing-subscriptions/#from-youtube

**Reply by gdwaynewarner** [reply]
Yes. I followed those instructions, and FreeTube doesn't seem to like my Firefox files.

**Reply by absidue** [reply]
@gdwaynewarner Have you tried clicking on the button and then selecting the files withe file picker that shows up, instead of dragging and dropping?

**Reply by gdwaynewarner** [reply]
Unfortunately, I don't know how to do that.

That is, I don't know how to bring up the File Picker.

**Reply by absidue** [reply]
@gdwaynewarner Open FreeTube's settings page, open the "Data Settings" section and click on the "Import Subscriptions" button.

**Reply by gdwaynewarner** [reply]
I did that. It would not let me click the file I need to import (watch-history.html).

**Reply by gdwaynewarner** [reply]
If it helps any, that file is about 50 megabytes in size.
